public/index.html

```diff
@@ -15,8 +15,20 @@
 
         <object id="floor1" type="image/svg+xml" data="assets/maps/ICF1Edit.svg"></object>
         <object id="floor2" type="image/svg+xml" data="assets/maps/ICF2Edit.svg"></object>
-
+        <div id="pins">
+        </div>
+        <!-- <div id="card" class="card">
+            <div class="additional">
+                <div class="event-card">
+                </div>
+                <div class="more-info">
+                </div>
+            </div>
+            <div class="general">
+            </div>
+        </div> -->
+<!-- <button type="button" name="button" onclick="getRoomLocation(111)">Toggle card</button> -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Let's strive to never use/import jQuery, especially if we're just using it to load a file (and I don't think we'll need floors.json anyways – see other comment)